### DISCUSSION TITLE
Add nuxt.js with SSR to an existing site

### DISCUSSION THREAD

**Discussion by yleclanche** [nuxt/nuxt#8541] [Questions (Nuxt 2)]
Hello,

I'd like to use nuxt.js on an existing site (not made with Vue) to make the search engine more dynamic.
So in this configuration, the main component would be included on a dedicated page which already contains header, footer... generated by a CMS.

How to make SSR and routing works in this configuration ? Is this even possible for SSR ?

Thanks.

**Comment by devlsh** [comment]
Hey @yleclanche,

Could you explain more about the architecture involved here? For example, how are these existing pages served that Nuxt understands?

**Reply by yleclanche** [reply]
It's a website of classified ads.
Each classified ad has a page and there is a search engine to filter them. The site is made with Django.
When a user goes to /cars/ford, it displays the search engine with preconfigured filters (cars category and Ford brand).

We would like to have the same thing but with Vue and SSR to handle the search engine part.
